lib/MathChangeTypes.js

```diff
@@ -7,26 +7,28 @@ module.exports = {
   COLLECT_LIKE_TERMS: 'Collect like terms',
   COLLECT_AND_COMBINE_LIKE_TERMS: 'Collect and combine like terms',
 
-  UNARY_MINUS_TO_NEG_ONE: 'A minus is just like a coefficient of -1',
+  UNARY_MINUS_TO_NEGATIVE_ONE: 'A minus is just like a coefficient of -1',
   ADD_COEFFICIENT_OF_ONE: 'No coefficient is the same a coefficient equal to 1',
   ADD_EXPONENT_OF_ONE: 'No exponent is the same as a coefficient of 1',
   GROUP_COEFFICIENTS: 'Group coefficients together',
   COLLECT_EXPONENTS: 'Sum up the exponents',
   MULTIPLY_COEFFICIENTS: 'Multiply coefficients together',
   ADD_POLYNOMIAL_TERMS: 'Add terms together',
-  MULT_POLYNOMIAL_TERMS: 'Multiply terms together',
+  MULTIPLY_POLYNOMIAL_TERMS: 'Multiply terms together',
   REARRANGE_COEFF: 'Move the coefficient in front',
+  REDUCE_EXPONENT_BY_ZERO: 'reduce-exponent-by-zero',
```

Comment:
these actually need real descriptions until Ahmed's change goes in, cause we'll be deploying it to prod probably